java/kotlin-extractor2/src/main/kotlin/utils/TypeResults.kt

```diff
@@ -14,21 +13,14 @@ OLD: KE1
  * arguments ("InnerClass<E>" or "OuterClass<ConcreteArgument>" or "OtherClass<? extends Bound>") or
  * an array ("componentShortName[]").
  */
-*/
 data class TypeResultGeneric<SignatureType, out LabelType : AnyDbType>(
     val id: Label<out LabelType>,
-    /*
-    OLD: KE1
-        val signature: SignatureType?,
-        val shortName: String
-    */
+    // OLD: KE1: val signature: SignatureType?,
+    val shortName: String
```

Comment:
Passing `shortName` around is restored (used to generate names for wildcard types, like "? extends Blah" where "Blah" is the shortName of the underlying type). I haven't looked into how we're using `signature` yet so the relevant arguments are still behind OLD/KE1 comments.